datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/KafkaBasedConnectorConfig.java

```diff
@@ -31,6 +31,7 @@
   public static final String CONFIG_RETRY_SLEEP_DURATION_MILLIS = "retrySleepDurationMs";
   public static final String CONFIG_PAUSE_PARTITION_ON_ERROR = "pausePartitionOnError";
   public static final String CONFIG_PAUSE_ERROR_PARTITION_DURATION_MILLIS = "pauseErrorPartitionDurationMs";
+  public static final String CONFIG_ENABLE_POLL_DURATION_MS_METRIC = "enablePollDurationMsMetric";
```

Comment:
nit: Looking at the naming convention used in this file, MS is named as MILLIS

Reply:
done